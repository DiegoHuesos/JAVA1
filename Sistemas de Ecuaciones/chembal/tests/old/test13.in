H2O + H^ + Cl^- + MnO4^- + Cl2 + Mn^2 + Cu + HNO3 + Cu(NO3)2 + NO + Na2O3 + Fe2O3 + FeCl3 + CO2 + NaCl + HCl
