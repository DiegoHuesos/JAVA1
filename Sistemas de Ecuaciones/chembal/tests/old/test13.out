8H2O + 5Cl2 + 2Mn^2 --> 16H^ + 10Cl^- + 2MnO4^-
2H^ + 2Cl^- + Cu(NO3)2 --> Cl2 + Cu + 2HNO3
6H^ + 6Cl^- + 2HNO3 --> 4H2O + 3Cl2 + 2NO
6H^ + 6Cl^- + Fe2O3 --> 3H2O + 2FeCl3
6H^ + 6Cl^- + Na2O3 --> 3H2O + 2Cl2 + 2NaCl
H^ + Cl^- --> HCl
